internal/resources/resource_instances_helper.go

```diff
@@ -216,6 +216,11 @@ func getInstanceDefaultSchema(isClone bool) *schema.Resource {
 							Optional:    true,
 							Description: "Asset tag",
 						},
+						"create_user": {
+							Type:        schema.TypeBool,
```

Comment:
I think this cant be updated post the VM creation. which means if the user tries to update this field, it shud delete the existing instance and create a new instance(forcenew=true) all together or shud we add a error message saying, this field cant be updated ? @gandharvas please share ur inputs here

Reply:
@manjunath-batakurki you are right, since this option is not supported post creation of the VM, it is necessary to set it as forceNew.